src/types/ErrorMessage.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorMessage {
+  NULL = '',
```

Comment:
Why do you need this value? Just use empty string
Comparing with empty string could be replaced with !string or just string( in conditions)